src/entity/posts/utils.ts

```diff
@@ -340,6 +341,49 @@ export const createSharePost = async (
   }
 };
 
+export const updateSharePost = async (
+  con: DataSource | EntityManager,
+  userId: string,
+  postId: string,
+  sourceId: string,
+  commentary: string | null,
+) => {
+  let strippedCommentary = commentary;
+
+  if (commentary?.length) {
+    strippedCommentary = commentary.trim();
+    await validateCommentary(strippedCommentary);
+  } else {
+    strippedCommentary = null;
+  }
```

Comment:
I would say maybe uniform this with the one in `validatePost`.
Ideally we have two extra functions there: `validateTitle` and `validateCommentary` the validatePost can then re-use this.

It's just to ensure all logic is the same accross posts.

Reply:
Ah I see this was existing, and evaluates a little different as it's technically a title, but different name.
Maybe we can add the trim stuff inside this `validateCommentary`?